src/PaymentElement.res

```diff
@@ -158,6 +158,19 @@ let make = (~cardProps, ~expiryProps, ~cvcProps, ~paymentType: CardThemeType.mod
     googlePayThirdPartySessionObj.sessionsToken,
     Gpay,
   )
+  let paypalPaymentMethodExperience = React.useMemo(() => {
```

Comment:
This code seems to be duplicated in PaymentUtils.res as well. Could you please move it to a common file

Reply:
Updated